pkg/agent/filestore/filestore.go

```diff
@@ -0,0 +1,166 @@
+// Copyright 2025 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package filestore
+
+import (
+	"bytes"
+	"encoding/gob"
+	"fmt"
+	"io"
+	"io/fs"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/afero"
+	"k8s.io/klog/v2"
+
+	"antrea.io/antrea/pkg/agent/interfacestore"
+)
+
+// Set it to NewMemMapFs as the file system may be not writable.
+// Change it to NewOsFs to evaluate performance when writing to disk.
+// It's for testing only.
+var newFS = afero.NewMemMapFs
```

Comment:
this variable doesn't seem to be really needed
you only use it for `NewFakeFileStore`, which should be moved to the test file anyway

Reply:
Refined.